RecoLocalCalo/HcalRecAlgos/interface/HLTv2.h

```diff
@@ -0,0 +1,63 @@
+#ifndef HLTv2_h
+#define HLTv2_h 1
```

Comment:
- Please use an appropriate file include guard, include the package and subsystem name
- rename the file and the class to something more Hcal* specific. If there is a chance this method becomes useful offline the "HLT" in the name will be very confusing. HcalDeterministicPulseFit may be a more appropriate name
- don't use ROOT-related typedefs (Float_t -> float; Double_t -> double)